### ISSUE TITLE
Add Skip to Content Link

### ISSUE THREAD

**Issue by slowbot** [bloom-housing/bloom#2184]
**What is this feature?**
There should be a Skip to Content link at the start of the page that allows screen reader users to skip tabbing through the navigation bar and be focused in the start of the main content area, to prevent a user from having to tab through the navigation on every single page visit if they aren't going to need it. We should add this on every page in the public site, we can probably add it to our layout or navigation component that wraps every page.

WCAG documentation: https://www.w3.org/TR/WCAG20-TECHS/G1.html

**Describe the solution you'd like**
Add a Skip to Content link that is focusable and directs user to content area

**What is the acceptance criteria/definition of done?**
- User should be able to tab into a Skip to Content link
- Taking action on the link should direct the user to the start of the content area

**Comment by slowbot** [comment]
@emilyjablonski For your review

**Comment by emilyjablonski** [comment]
@slowbot I'm assuming we want this on every page in public, do we want it on partners as well?

**Comment by slowbot** [comment]
@emilyjablonski Yes, all pages in public. Partners is lower priority but if its a quick add I say we do it.

**Comment by jaredcwhite** [comment]
Glad to find this issue here, I was working on adding a skip link on another project and then wondered if Bloom had it. I'd be happy to take this on.

**Comment by slowbot** [comment]
Addressed in https://github.com/bloom-housing/bloom/issues/3270
